packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -105,19 +111,14 @@ export default class ButtonCore extends React.Component<Props> {
                     icon && sharedStyles.textWithIcon,
                     spinner && sharedStyles.hiddenText,
                     kind === "tertiary" && sharedStyles.textWithFocus,
-                    // apply focus effect on the label instead
+                    // apply press/hover effects on the label
                     kind === "tertiary" &&
                         !disabled &&
                         (pressed
-                            ? buttonStyles.active
-                            : (hovered || focused) && buttonStyles.focus),
-                    kind === "tertiary" &&
-                        disabled &&
-                        focused && [
-                            buttonStyles.focus,
-                            buttonStyles.disabledFocus,
-                        ],
+                            ? [buttonStyles.hover, buttonStyles.active]
+                            : hovered && buttonStyles.hover),
                 ]}
+                testId="button-inner-label"
```

Comment:
**suggestion:** I'd recommend using a composed `testId` value. In this case works fine, but if for any reason we have multiple `Button` instances in `webapp`, then identifying this test-id for a specific button instance will be more complex.

As we already have a `testId` prop in `Button`, we can probably do something like:

```suggestion
                testId={testId ? `${testId}-inner-label` : undefined}
```

Then, you'd need to assign a `testId` value in the `Button` instance included in the interaction test for this to work.